packages/amazon-sumerian-hosts-core/src/core/GestureFeature.js

```diff
@@ -6,6 +6,9 @@ import AbstractHostFeature from './AbstractHostFeature';
 import Deferred from './Deferred';
 import Utils from './Utils';
 
+/**
+ * @constant
+ */
 const DefaultGestureWords = {
```

Comment:
This is not showing up in the API docs even though it is one of this module's exports.

Reply:
Can you help me understand what "Globals" means in this context? Based on how we're packaging the code, how would I access that constant? Would it be `DefaultGestureWords` (with no qualifier...a true global)? `HOST.DefaultGestureWords`? `HOST.GestureFeature.DefaultGestureWords`?

Reply:
I figured it out. I would use `HOST.DefaultGestureWords`.

OK. You can ignore this comment then.

Reply:
I mean this DefaultGestureWords shows up in the API docs: **Global** section. If I understand right, you can use Host.DefaultGestureWords.